packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -189,6 +190,7 @@ export class VirtualNode extends VirtualNodeBase {
     props.listeners?.forEach(listener => this.addListener(listener));
     const accessLogging = props.accessLog?.bind(this);
     const serviceDiscovery = props.serviceDiscovery?.bind(this);
+    const backendDefaultsTlsClientPolicy = props.backendDefaults?.tlsClientPolicy;
```

Comment:
Since this variable is used exactly once, I don't think it's very valuable. I would inline it.

Reply:
Inlined it.